sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_with_batch_transform.ipynb

```diff
@@ -148,7 +150,7 @@
     "\n",
     "Now we can perform some inference with the model we've trained by running a batch transform job. The request handling behavior of the Endpoint deployed during the transform job is determined by the `mnist.py` script.\n",
     "\n",
-    "For demonstration purposes, we will be using an image of a '7' that's already saved in S3:"
+    "For demonstration purposes, we're going to use input data that contains ten MNIST images, located in the public SageMaker sample data S3 bucket. To create the batch transform job, we simply call `transform()` on our transformer with information about the input data."
```

Comment:
Is there a reason we're using such a small number of images considering this is batch transform?

Reply:
Mostly I just didn't want it to take too long to run.

Reply:
Since most of the batch transform run time is spent spinning up the instances, if it's reasonably easy, it might make sense to move to a larger number of images.

Reply:
Okay, bumped it up to 1000 images. Thought about going higher, but it took awhile to upload 5000 images to S3 for one region.